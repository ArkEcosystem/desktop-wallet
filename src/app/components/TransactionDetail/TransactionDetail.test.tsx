import { render } from "@testing-library/react";
import React from "react";

import { TransactionDetail } from "./";

describe("TransactionDetail", () => {
	it("should render", () => {
		const { container } = render(<TransactionDetail label="Test">test</TransactionDetail>);
		expect(container).toMatchSnapshot();
	});

	it("should render without border", () => {
		const { container } = render(
			<TransactionDetail label="Test" border={false}>
				test
			</TransactionDetail>,
		);
		expect(container).toMatchSnapshot();
	});

	it("should render without padding", () => {
		const { container } = render(
			<TransactionDetail label="Test" padding={false}>
				test
			</TransactionDetail>,
		);
		expect(container).toMatchSnapshot();
	});
});
