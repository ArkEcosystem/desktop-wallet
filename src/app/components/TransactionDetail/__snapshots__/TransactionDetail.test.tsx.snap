// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionDetail should render 1`] = `
<div>
  <div
    class="flex items-center py-6 border-t border-dashed border-theme-neutral-300 "
  >
    <div
      class="flex-1 space-y-2"
    >
      <div
        class="text-sm font-semibold text-theme-neutral"
      >
        Test
      </div>
      <div
        class="font-semibold"
      >
        test
      </div>
    </div>
  </div>
</div>
`;

exports[`TransactionDetail should render with extra children 1`] = `
<div>
  <div
    class="flex items-center true border-t border-dashed border-theme-neutral-300 "
  >
    <div
      class="flex-1 space-y-2"
    >
      <div
        class="text-sm font-semibold text-theme-neutral"
      >
        Test
      </div>
      <div
        class="font-semibold"
      >
        test
      </div>
    </div>
    <div
      data-testid="TEST_CHILD"
    />
  </div>
</div>
`;

exports[`TransactionDetail should render without border 1`] = `
<div>
  <div
    class="flex items-center py-6 true "
  >
    <div
      class="flex-1 space-y-2"
    >
      <div
        class="text-sm font-semibold text-theme-neutral"
      >
        Test
      </div>
      <div
        class="font-semibold"
      >
        test
      </div>
    </div>
  </div>
</div>
`;

exports[`TransactionDetail should render without padding 1`] = `
<div>
  <div
    class="flex items-center true border-t border-dashed border-theme-neutral-300 "
  >
    <div
      class="flex-1 space-y-2"
    >
      <div
        class="text-sm font-semibold text-theme-neutral"
      >
        Test
      </div>
      <div
        class="font-semibold"
      >
        test
      </div>
    </div>
  </div>
</div>
`;
